internal/config/verifier.go

```diff
@@ -2,24 +2,20 @@ package config
 
 import (
 	"fmt"
-	"os"
 
+	zk "github.com/rarimo/zkverifier-kit"
 	"gitlab.com/distributed_lab/figure/v3"
 	"gitlab.com/distributed_lab/kit/kv"
 )
 
-type VerifierConfig struct {
-	VerificationKeys    map[string][]byte
-	AllowedAge          int
-	AllowedCitizenships []interface{} // more convenient to use for validation, replace on need
-}
+const proofEventIDValue = "304358862882731539112827930982999386691702727710421481944329166126417129570"
 
-func (c *Config) Verifier() *VerifierConfig {
+func (c *Config) Verifier() *zk.Verifier {
 	return c.verifier.Do(func() interface{} {
 		var cfg struct {
-			VerificationKeysPaths map[string]string `fig:"verification_keys_paths,required"`
-			AllowedAge            int               `fig:"allowed_age,required"`
-			AllowedCitizenships   []string          `fig:"allowed_citizenships,required"`
+			VerificationKeyPath string   `fig:"verification_key_path,required"`
```

Comment:
Is there one main curcuit that manges all signatures (ecdsa, sha etc.)? Or later we will add one more key?

Reply:
At this stage we have one key for everything, as we always had (verification_key.json). I suggest we work around when different keys are required